<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignVerifyDialog</class>
 <widget class="QDialog" name="SignVerifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign / Verify</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Signature</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPlainTextEdit" name="message"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="address"/>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_Sign">
       <property name="text">
        <string>Sign</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Verify">
       <property name="text">
        <string>Verify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Copy">
       <property name="text">
        <string>Copy to clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="signature"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
